.github/workflows/ci-deploy.yml

```diff
@@ -77,19 +77,16 @@ jobs:
           --tag $ECR_REGISTRY/$ECR_REPOSITORY:$BRANCH \
           --file ./Dockerfile ./
 
-    - name: Kick off Terraform deploy in sysops/ if not Live environment
+    - name: Kick off Terraform deploy in sysops/
       id: sysops-deploy
       run: |
-        BRANCH=$(echo "${GITHUB_REF#refs/*/}" | sed 's/.*\///')
-        if [[ "$BRANCH" != master ]]; then
-          curl \
+        curl \
           -X POST \
           -H "Accept: application/vnd.github+json" \
           -H "Authorization: Bearer ${{ secrets.SYSOPS_RW_GITHUB_TOKEN }}" \
           -H "X-GitHub-Api-Version: 2022-11-28" \
           https://api.github.com/repos/meedan/sysops/actions/workflows/deploy_${{ github.event.repository.name }}.yml/dispatches \
-          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "branch": "'$BRANCH'"}}'
-        fi
+          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "workspace": "qa"}}'
```

Comment:
How would we keep the mapping in sysops itself? (When sysops is fired I only get a commit SHA otherwise, and I can't tell from that which environment is intended.)

I might be missing a trick to knowing context for a workflow dispatch...

Reply:
you can pass any variables you want to a dispatch, this is in the gha workflow docs